manubot/cite/doi.py

```diff
@@ -146,6 +146,16 @@ def get_doi_csl_item_crosscite(doi: str):
         raise error
 
 
+get_doi_csl_item_datacite = partial(
+    get_doi_csl_item_negotiation,
+    content_negotiation_url=content_negotiation_url_datacite,
+)
+get_doi_csl_item_crosscite = partial(
+    get_doi_csl_item_negotiation,
+    content_negotiation_url=content_negotiation_url_crosscite,
+)
```

Comment:
`partial` introduces a bug in

https://github.com/manubot/manubot/blob/b8a00c0f33fe7eafed3515154a190f3aefd19182/manubot/cite/doi.py#L203-L212

https://github.com/manubot/manubot/runs/4748725137?check_suite_focus=true#step:6:1113

Reply:
removing partial in https://github.com/manubot/manubot/pull/312